### ISSUE TITLE
rake commands not working

### ISSUE THREAD

**Issue by mbrennan** [james2m/seedbank#1]
Hello, in my Rails 3 project, I added seedbank to my Gemfile and did a bundle install.  bundle show seedbank shows the gem installed.  rake -T shows seedbank commands like db:seed:development .  However, when I run rake db:seed:development, it doesn't run the scripts in my development directory.  Also, if I try a specific script like rake db:seed:development:users, the command line shows:

```
rake aborted!
Don't know how to build task 'db:seed:development:users'
```

**Comment by jjercx** [comment]
Same here!

**Comment by ivanovaleksey** [comment]
@jjercx it is definitely something wrong with your environment. As James wrote earlier could you show `Gemfile.lock` and the content of `db/seeds`?

**Comment by jjercx** [comment]
The problem was I deleted db/seeds.rb. `touch db/seeds.rb` fixed it for me. Thanks.

**Comment by james2m** [comment]
This is fixed on master and 0.5.0.pre.
